VideoLocker/AndroidManifest.xml

```diff
@@ -48,7 +48,8 @@
             android:name="org.edx.mobile.view.SplashActivity"
             android:label="@string/app_name"
             android:screenOrientation="portrait"
-            android:theme="@style/Theme.CustomPreview">
+            android:theme="@android:style/Theme.NoDisplay"
+            android:excludeFromRecents="true">
```

Comment:
I think this attribute can be removed, as it seems to be causing the whole app to be excluded. Since the splash screen is immediately finished, I don't think it will displayed in the recents in any case.